alex/applications/PublicTransportInfoCS/data/train_names.expanded.txt

```diff
@@ -0,0 +1,178 @@
+albert einstein	albert einstein
```

Comment:
What's the difference between "expanded" and "plain" train names? Are these supposed to be inflected in all cases?